<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>temperatureCalcWindow</class>
 <widget class="QWidget" name="temperatureCalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>temperature calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lbl_voltage_lbl">
     <property name="text">
      <string>voltage:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QComboBox" name="cob_voltage"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lbl_current_lbl">
     <property name="text">
      <string>current:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QComboBox" name="cob_current"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_status_lbl">
     <property name="text">
      <string>status:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="lbl_status">
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>pausing</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QPushButton" name="btn_startStop">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>run</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="lbl_showShare">
     <property name="text">
      <string>show / share:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QCheckBox" name="chb_linear">
     <property name="text">
      <string>linear</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QCheckBox" name="chb_quadratic">
     <property name="text">
      <string>quadratic</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QGroupBox" name="grp_linear">
     <property name="title">
      <string>linear</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="lbl_initValue">
        <property name="text">
         <string>measure R at T[K] = </string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="dsb_initTemperature">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>400.000000000000000</double>
        </property>
        <property name="value">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btn_measureInitValue">
        <property name="text">
         <string>measure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="lbl_linearInitValues">
        <property name="text">
         <string>(no init values set/measured yet)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="lbl_alpha">
        <property name="text">
         <string>α[1/K] = 10⁽⁻³⁾*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QDoubleSpinBox" name="dsb_alpha">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>4.400000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_linearCalcTemperatureCelsius_lbl">
        <property name="text">
         <string>T[°C] = </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="txt_linearCalcTemperatureCelsius">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="lbl_linearCalcTemperatureKelvin_lbl">
        <property name="text">
         <string>T[K] =</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="txt_linearCalcTemperatureKelvin">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QGroupBox" name="grp_quadratic">
     <property name="title">
      <string>quadratic</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_quadraticCalcTemperatureCelsius_lbl">
        <property name="text">
         <string>T[°C] = </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_quadraticCalcTemperatureCelsius">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lbl_quadraticCalcTemperatureKelvin_lbl">
        <property name="text">
         <string>T[K] =</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="txt_quatraticCalcTemperatureKelvin">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_diameter_lbl">
        <property name="text">
         <string>d[mm]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="dsb_diameter">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_length_lbl">
        <property name="text">
         <string>l[mm]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="dsb_length">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
